client/src/components/Login.jsx

```diff
@@ -1,19 +1,51 @@
 import React from 'react';
+import axios from 'axios';
 
 class Login extends React.Component {
   constructor(props) {
     super(props);
+
+    this.state = {username: '', password: ''};
+  }
+
+  onLoginSubmit () {
+    let userInfo = {username: this.state.username, password: this.state.password};
+    axios({
+      url: '/login',
+      method: 'POST',
+      data: userInfo,
+    })
+      .then (
+        console.log('Login submission posted')
+      )
+      .catch ((error) => {
+        console.log('An error occurred inside of the onLoginSubmit method' + error);
+      });
+  }
+
+  handleClickUser (event) {
```

Comment:
You could create 1 function, like "handleChange" and use the event.target.name to define which state property you're changing.